libvirt/utils_domain_def.go

```diff
@@ -21,20 +21,39 @@ func getGuestForArchType(caps libvirtxml.Caps, arch string, virttype string) (li
 	return libvirtxml.CapsGuest{}, fmt.Errorf("[DEBUG] Could not find any guests for architecure type %s/%s", virttype, arch)
 }
 
+func huntForMachine(machines []libvirtxml.CapsGuestMachine, targetmachine string) string {
```

Comment:
`huntForMachine` is rather unusual name. 

I would call it `lookupForMachine`.

Reply:
I settled on `lookupMachine`.  Let me know if that is satisfactory.